client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -195,14 +202,22 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
   };
 
   const login = async (username: string, password: string) => {
-    setSkipRequest(skipNoStoreRequests);
     const { token, error } = await mutateAsync({ username, password });
     setHeader('Authorization', `Bearer ${token}`);
     const store = await getStore(token);
+    const permissions = await getPermissions({
+      storeId: store?.id || '',
+      token,
+    });
+    setSkipRequest(skipNoStoreRequests);
     const authCookie = {
       store,
       token,
-      user: { id: '', name: username },
+      user: {
+        id: '',
+        name: username,
+        permissions: permissions?.nodes?.[0]?.permissions || [],
```

Comment:
just storing permissions for the current store to make usage easier 🤷

Reply:
Is it updated when changing the store?

Reply:
ignore it does

Reply:
could you call setStore here after setting the basic cookie?

Reply:
good thought! hadn't considered that. will see - that should reduce some duplication.